cl/alerts/tasks.py

```diff
@@ -100,6 +117,26 @@ def send_docket_alert(d_pk: int, since: datetime) -> None:
     connection = get_connection()
     connection.send_messages(messages)
 
+    # Notify all the webhooks available for the users subscribed
+    for user in users:
+        for webhook in user.webhooks.all():  # type: ignore
+            if webhook.enabled:
+                post_content = make_docket_alert_webhook_payload(
+                    new_des, webhook
+                )
+                response = requests.post(
+                    webhook.url, data=post_content, timeout=30
+                )
+                WebhookEvent.objects.create(
+                    webhook=webhook,
+                    status_code=response.status_code,
+                    content=post_content,
+                    response=response.text,
+                )
+                if not response.ok:
+                    webhook.failure_count = webhook.failure_count + 1
+                    webhook.save()
```

Comment:
This lacks retries, which will be a problem. If you put it in a celery task, that'll accomplish retries though.